manage

```diff
@@ -692,11 +692,45 @@ themes.oscar() {
 }
 
 themes.simple() {
-    build_msg GRUNT "theme: simple"
-    npm --prefix searx/static/themes/simple run build
+    local src="searx/static/themes/simple/src"
+    local static="searx/static/themes/simple"
+    (   set -e
+        convert_if_newer "$src/svg/searxng-wordmark.svg" "$static/img/favicon.png" \
+                         -transparent white -resize 64x64
```

Comment:
> The favicon.png does not have an alpha channel:

@dalf thanks a lot for attention! ... I'm very sorry for the mess .. it is not ImageMagick what is broken .. it is me 🤦🏻 

Its now fixed ...

![grafik](https://user-images.githubusercontent.com/554536/141513942-ec987d71-6286-4e3a-bc90-4905b12b2b9a.png)

Reply:
The favicon is fixed now :+1: :
![image](https://user-images.githubusercontent.com/38733246/141597424-592a48f1-4ab1-4afd-97b4-5f55b81155df.png)